mmv1/templates/terraform/examples/cloud_run_service_secret_environment_variables.tf.erb

```diff
@@ -1,35 +1,23 @@
-data "google_project" "project" {
-  provider = google-beta
```

Comment:
You're still going to want this empty block, I bet.

Reply:
Done. Trying to run the tests on my local machine for ga and beta.

Reply:
Tests ran successfully both for ga and beta.

for ga:

=== RUN   TestAccCloudRunService_secretEnvironmentVariable
=== PAUSE TestAccCloudRunService_secretEnvironmentVariable
=== CONT  TestAccCloudRunService_secretEnvironmentVariable
--- PASS: TestAccCloudRunService_secretEnvironmentVariable (76.99s)
PASS

=== RUN   TestAccCloudRunService_secretVolume
=== PAUSE TestAccCloudRunService_secretVolume
=== CONT  TestAccCloudRunService_secretVolume
--- PASS: TestAccCloudRunService_secretVolume (74.17s)
PASS

for beta:

=== RUN   TestAccCloudRunService_secretEnvironmentVariable
=== PAUSE TestAccCloudRunService_secretEnvironmentVariable
=== CONT  TestAccCloudRunService_secretEnvironmentVariable
--- PASS: TestAccCloudRunService_secretEnvironmentVariable (78.51s)
PASS

=== RUN   TestAccCloudRunService_secretVolume
=== PAUSE TestAccCloudRunService_secretVolume
=== CONT  TestAccCloudRunService_secretVolume
--- PASS: TestAccCloudRunService_secretVolume (85.45s)
PASS